CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Added
+- ([#325](https://github.com/demos-europe/demosplan-ui/pull/325)) **BREAKING** remove spacing classes from DpAnonymizeText. ([@ahmad-demos](https://github.com/ahmad-demos))
```

Comment:
In other places, this is also not capitalized. But after it, a colon appears and the next word is written as if a sentence would start:

```suggestion
- ([#325](https://github.com/demos-europe/demosplan-ui/pull/325)) **Breaking:** Remove spacing classes from DpAnonymizeText ([@ahmad-demos](https://github.com/ahmad-demos))
```